dynawo/sources/Common/Dictionaries/DYNTimeline_en_GB.dic

```diff
@@ -40,6 +40,10 @@ PhaseShifterBelowMin      =             Phase-shifter : below minimum allowed va
 PhaseShifterBelowMax      =             Phase-shifter : below maximum allowed value
 PhaseShifterBelowStop     =             Phase-shifter : back to acceptable value
 PhaseShifterWithinInterval=             Phase-shifter : back to acceptable value
+// --> Phase-shifter Lock
```

Comment:
I think we could choose between lock and block and stick to it. I prefer Block/Blocking (like it has been changed for the tap changer blocking).

Reply:
Blocking